app/policies/public_activity/activity_policy.rb

```diff
@@ -37,7 +37,9 @@ def resolve
                "(team_users.user_id = ? OR namespaces.public = ?)",
                "Repository", user.id, true)
 
-      team_activities.union_all(namespace_activities).union_all(repository_activities).distinct
+      team_activities.union_all(namespace_activities)
```

Comment:
I added a policy for comment-activities and after that the line was too long. Later I forgot to restore the orginally line.